forecast_tools.R

```diff
@@ -86,8 +86,8 @@ plot_data = function(data) {
     text = getURL(
       "https://raw.githubusercontent.com/weecology/PortalData/master/Rodents/moon_dates.csv"))
   target_moon=unique(data$NewMoonNumber)
-  period_code = dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
-    dplyr::select(Period)
+  period_code = as.integer(dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
+    dplyr::select(Period))
```

Comment:
Ah, yep, sorry, one more try:

```
git push -f origin repair_plot_data_func
```

The `-f` "forces" the push, which is needed because by using `git commit --amend` we rewrote the history and git wants to make sure that we know what we're doing. If instead of using `--amend` you'd added a new commit it would have pushed fine.